doc-surrealdb_versioned_docs/version-1.x/introduction/start.mdx

```diff
@@ -63,7 +63,7 @@ For the sake of this guide, we recommend using a sandbox connection. Once you ha
 
 </TabItem>
 <TabItem value="CLI" label="Using CLI">
-Once you have your [database running](/docs/surrealdb/cli/start), you can use the SurrealDB CLI to run your queries. To do so, run the following commands:
+Once you have your [database running](/docs/surrealdb/cli/start), you can use the [SurrealDB CLI](https://surrealdb.com/docs/surrealdb/cli/) to run your queries. To do so, run the following commands:
```

Comment:
We use relative links to avoid a refresh. 
```suggestion
Once you have your [database running](/docs/surrealdb/cli/start), you can use the [SurrealDB CLI](/docs/surrealdb/cli/) to run your queries. To do so, run the following commands:
```